Sources/SpeziQuestionnaire/TimedWalkView.swift

```diff
@@ -0,0 +1,91 @@
+//
+// This source file is part of the Stanford Spezi open-source project
+//
+// SPDX-FileCopyrightText: 2022 Stanford University and the project authors (see CONTRIBUTORS.md)
+//
+// SPDX-License-Identifier: MIT
+//
+
+import FHIRQuestionnaires
+import ModelsR4
+import ResearchKit
+import ResearchKitOnFHIR
+import SwiftUI
+
+
+/// Renders a FHIR `Questionnaire`.
+public struct TimedWalkView: View {
+    @EnvironmentObject private var questionnaireDataSource: QuestionnaireDataSource
+    
+    @Binding private var isPresented: Bool
+    
+    private let identifier: String?
+    private let distanceInMeters: Double?
+    private let timeLimit: TimeInterval?
+    private let turnAroundTimeLimit: TimeInterval?
+    private let timedWalkResponse: ((ORKTimedWalkResult) async -> Void)?
+
+        
+    public var body: some View {
+        if let task = createTask() {
+            ORKOrderedTaskView(
+                tasks: task,
+                isPresented: $isPresented,
+                timedWalkResponse: { response in
+                    await timedWalkResponse?(response)
+                    await questionnaireDataSource.add(response)
```

Comment:
What is the datasource for? To store responses? How many responses are returned by timedWalk? I thought it was only 1 response.